### ISSUE TITLE
[Feature Request] change in the settings of the background/popup list the "sound settings" by the equalizer

### ISSUE THREAD

**Issue by speednroll** [TeamNewPipe/NewPipe#1055]
would it not be easier for the equalizer to appear as in other players instead of just sending you to the sound settings in the options in the list of videos?
![](https://framapic.org/V9dnVU0bUfDG/2DiSkhcRicUB.png)

**Comment by theScrabi** [comment]
Not all phones have an equalizer app do the?

**Comment by speednroll** [comment]
It is not my case, I use a stock rom. the equalizer it is hidden, I can only access it from spotify and other music players and the settings are only saved for that application

**Comment by speednroll** [comment]
Look at this: for example, in spotify, when you tap the options of the equalizer sends me directly to the system's equalizer, which only configures the options for that application.

![](https://framapic.org/pIPsnEozPdeu/0PPzUrbZZd2R.png)

**Comment by theScrabi** [comment]
@speednroll could you please adjust the size of an image if you post them here.

**Comment by speednroll** [comment]
I'm sorry, but I'm on the phone. How do I adjust them? 😑

**Comment by justanidea** [comment]
With any picture editor 😅

**Comment by speednroll** [comment]
Of course, I'll do it when I'm in front of a computer. I thought there would be some code here to shrink the image without having to edit it :stuck_out_tongue_winking_eye:
